nautobot/utilities/tests/test_filters.py

```diff
@@ -671,9 +671,9 @@ def test_tag_filter(self):
 
     def test_tree_node_multiple_choice_filter(self):
         self.assertIsInstance(self.filters["treeforeignkeyfield"], TreeNodeMultipleChoiceFilter)
-        self.assertEqual(self.filters["treeforeignkeyfield"].lookup_expr, "in")
+        self.assertEqual(self.filters["treeforeignkeyfield"].lookup_expr, "exact")
         self.assertEqual(self.filters["treeforeignkeyfield"].exclude, False)
-        self.assertEqual(self.filters["treeforeignkeyfield__n"].lookup_expr, "in")
+        self.assertEqual(self.filters["treeforeignkeyfield__n"].lookup_expr, "exact")
         self.assertEqual(self.filters["treeforeignkeyfield__n"].exclude, True)
```

Comment:
Interestingly, in NetBox, `treeforeignkeyfield` is expected to be `exact` but `treeforeignkeyfield__n` is expected to be `in`. What's the functional impact of this change?

Reply:
I fixed it.